src/System.Linq.Expressions/src/System/Linq/Expressions/Strings.cs

```diff
@@ -163,6 +168,114 @@ internal static string VariableMustNotBeByRef(object p0, object p1)
         }
 
         /// <summary>
+        /// A string like "Collection is read-only."
+        /// </summary>
+        internal static string CollectionReadOnly
```

Comment:
We converted a bunch of these to use expression-bodied members in a prior PR; for consistency, we should do that here too.